amundsen_application/__init__.py

```diff
@@ -43,10 +43,7 @@ def create_app(config_module_class: str, template_folder: str = None) -> Flask:
     tmpl_dir = template_folder if template_folder else os.path.join(PROJECT_ROOT, static_dir, 'dist/templates')
     app = app_wrapper_class(__name__, static_folder=static_dir, template_folder=tmpl_dir, **args)
 
-    """ Support for importing a custom config class """
-    config_module_class = \
-        os.getenv('FRONTEND_SVC_CONFIG_MODULE_CLASS') or config_module_class
```

Comment:
maybe just put `os.getenv('FRONTEND_SVC_CONFIG_MODULE_CLASS')` for the method's config_module_class default input value

Reply:
@feng-tao Updated to use this style. There are two caveats:
* Can't use default input values, since that's executed at definition time, not per-invocation
* The call-site still needs to check the env var, to pass in the fallback static default. (this could be moved into the function itself, but didn't feel right to have `amundsen_application.config.TestConfig` in `create_app`, open to feedback of course).